public/components/trace_analytics/requests/traces_request_handler.ts

```diff
@@ -141,6 +152,13 @@ export const handleTracesRequest = async (
       const percentileRanges =
         percentileRangesResult.status === 'fulfilled' ? percentileRangesResult.value : {};
       const response = responseResult.value;
+
+      // Check if the mode hasn't been set first
+      // if (mode === 'jaeger' && !response?.aggregations?.service_type?.buckets) {
+      //   console.warn('No traces or aggregations found.');
+      //   return [];
+      // }
```

Comment:
is this needed?

Reply:
Removed, became unnecessary after the url mode implementation.